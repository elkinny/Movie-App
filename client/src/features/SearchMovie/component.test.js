import SearchMovieComponent from './component';
import React from 'react';
import { shallow } from 'enzyme';

describe('SearchMovieComponent: ', () => {
  it('1. To match snapshot', () => {
    const props = {
      handleSubmit: () => {},
      handleToggle: () => {},
      moviesCount: 1,
      movies: [],
      searchToggleValue: 'searchToggleValue',
      sortToggleValue: 'sortToggleValue',
      formComponentRef: () => {},
    };
    const component = shallow(<SearchMovieComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
