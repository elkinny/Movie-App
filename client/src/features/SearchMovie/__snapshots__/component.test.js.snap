// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMovieComponent:  1. To match snapshot 1`] = `
<Fragment>
  <SearchFormContainer
    handleSubmit={[Function]}
    handleToggle={[Function]}
    searchToggleValue="searchToggleValue"
  />
  <SubHeader>
    <span>
      1 films are found
    </span>
    <Toggle
      labels={
        Array [
          "vote_average",
          "release_date",
        ]
      }
      onChange={[Function]}
      text="Sort by:"
      type="sort"
      value={true}
    />
  </SubHeader>
  <MoviesListComponent
    movies={Array []}
  />
</Fragment>
`;
